HW_15/HW_15_Logger_1.py

```diff
@@ -0,0 +1,44 @@
+# 12. Logger
+import logging
+
+logging.basicConfig(
+    filename='user_actions.log',
+    level=logging.DEBUG,
+    format='%(asctime)s - %(levelname)s - %(message)s',
+)
+
+
+def log_action(message, level='INFO'):
+    if level == 'INFO':
+        logging.info(message)
+    elif level == 'ERROR':
+        logging.error(message)
+
+
+def user_actions():
+    while True:
+        print("\nВыберите действие:")
+        print("1 - Добавить товар в корзину")
+        print("2 - Оформить заказ")
+        print("3 - Выйти из программы")
+
+        action = input("Ваш выбор: ")
+
+        if action == "1":
+            product = input("Введите название товара для добавления в корзину: ")
+            log_action(f"Пользователь добавил товар '{product}' в корзину.")
```

Comment:
You don't event need a special function here:

```suggestion
            logging.info(f"Пользователь добавил товар '{product}' в корзину.")
            ....
            logging.debug("Пользователь оформил заказ.")
```